### PR TITLE
docs(aio): add accessibility guide (a11y)

### PR DESCRIPTION
**Pull request by Foxandxss**
Tasks to do before merge:

- [x] Fix plunker link
- [x] Fix plunker demo 
- [x] Remove hardcoded toc and double check that the generated one is nice
- [x] Remove all mentions of "Angular 2" (prose and code)
- [x] Fix links pointing to other pages
- [x] Remove module.id from examples
- [x] Fix some examples headers
- [x] Check why the screenshots of the app are different from the final example (comment below)
- [x] Proper read
- [x] Resize images but leave some to be opened on click
- [x] Fix e2e tests
- [x] There needs to be glyphicons, but we don't have them
- [x] In some inner routes, after refresh, it doesn't work (¿? @brandonroberts)
- [x] Ward's review
- [ ] Confirm placement at the navigation.json

The screenshots are different from the final app. Code was modified after the screenshots were made.

### CODE REVIEW

File: aio/content/guide/a11y.md [hunk 1]
```diff
@@ -0,0 +1,1963 @@
...
+- tree
+- treegrid
+
+
+<div class="l-sub-section">
+
+
+
+Visit the `W3C` to read more about
+[Widget Roles](https://www.w3.org/TR/wai-aria/roles#widget_roles) and their
```
Comment (gkalpak) [inline aio/content/guide/a11y.md]: Again, this URL does not exists any more. https://www.w3.org/TR/wai-aria/#usage_intro seems relevant.

File: aio/content/guide/a11y.md [hunk 2]
```diff
@@ -0,0 +1,1963 @@
...
+    <img src="generated/images/guide/a11y/Chrome-experimental-a11y-inspector-unlabeled-input-700w.png" alt="Chrome accessibility inspector showing incorrectly labeled input"></img>
+  </figure>
+</a>
+
+
+#### OS X Safari 
+
+To activate accessibility inspection in Safari, 
+activate Web Inspector. Follow
+the steps in the [Enabling Web Inspector](https://developer.apple.com/library/mac/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/GettingStarted/GettingStarted.html)
```
Comment (gkalpak) [inline aio/content/guide/a11y.md]: This URL does not valid any more.

**Review by gkalpak — CHANGES_REQUESTED** [review]
This is amazing 👍 💯 ❤️

In addition to a few minor comments I left, a couple of other things we might want to do (but not necessarily blocking the PR):

- Get rid of the bootstrap-like classes/styles (they are kind of confusing imo) - or at least explain why we use them and what these classes do.
- Split the guide up into smaller ones.
- In several places throughout the guide, it is mentioned that something _"will be discussed later"_. It would be nice to have links to where it is actually discussed.

**Comment by marcysutton** [comment]
Still really looking forward to this one!

**Comment by gkalpak** [comment]
cc @jenniferfell

**Comment by marcysutton** [comment]
Can someone check up on this PR? It's been open in some form for literally years. The fact that Angular.io doesn't have an accessibility guide is really sad, and it contributes to the overall opinion that Angular is harder to get accessibility right. That isn't necessarily true, but people need more docs and information to know how to do it.

**Comment by petebacondarwin** [comment]
This PR has languished a long time @aikidave. I see that we now have an [a11y page](https://next.angular.io/guide/accessibility), although somewhat terser than what is in this PR. Should we just close this one and move on?

**Comment by Foxandxss** [comment]
Seeing the docs direction, the old this PR is and how much work it needs to be up to date, I guess we should move on.

If we consider that a person who comes looking for a11y knows about the subject, the current docs is enough-ish since it speaks about Angular specific.

This PR is more focused on giving information to someone who needs to do a11y but has no idea where to start.

Seeing the direction the docs have, it might be wise to move on.

**Comment by petebacondarwin** [comment]
> Seeing the docs direction, the old this PR is and how much work it needs to be up to date, I guess we should move on.

😢 

Thanks for commenting @Foxandxss - hope you are well.

Closing... @aikidave - if you would like to resurrect this PR please reopen.

**Comment by aikidave** [comment]
I appreciate all the work that's gone into this! I think closing the PR is the right course of action. However, I'm also happy to work with anyone who would like to help Angular developers create accessible applications. @Foxandxss thank you so much, and please don't hesitate to reach out to me to discuss this!

**Pull request closed without merging**